spec/models/filters/criteria/filter_for_dv_currently_fleeing_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative 'shared_filter_criteria_context'
+
+RSpec.describe Filters::Criteria::FilterForDvCurrentlyFleeing do
+  include_context 'filter criteria setup'
+
+  let(:currently_fleeing) { [1] } # Yes, currently fleeing
+  let(:filter) do
+    ::Filters::FilterBase.new(
+      user_id: user.id,
+      currently_fleeing: currently_fleeing,
+      start: start_date,
+      end: end_date,
+    )
+  end
+  let(:criteria) { described_class.new(input: filter, config: config) }
+
+  # Create enrollments with different currently fleeing statuses
+  let!(:enrollments) do
+    fleeing_statuses = [1, 0, 2] # Yes, No, Client doesn't know
```

Comment:
Doesn't know should be 8 throughout.
```suggestion
    fleeing_statuses = [1, 0, 8] # Yes, No, Client doesn't know
```

Reply:
thanks!